cranelift-codegen/meta/src/cdsl/ast.rs

```diff
@@ -168,6 +171,13 @@ impl Literal {
         }
     }
 
+    pub fn empty_vararg(kind: &OperandKind) -> Self {
+        Self {
+            kind: kind.clone(),
```

Comment:
Hmm, it seems it doesn't even need a kind, since it always generates the same literal (making the kind optional would be unsatisfactory, though)? Or we could reuse the "constant" kind for the immediate VarArgs?

Reply:
An alternative would be to make the `kind` field optional, or make it a field of each of the `LiteralValue` enumerated values.

Another option would be to create the kind field within this function instead of reusing the immediate value.

Reply:
ok, I will do that.